src/box/box.cc

```diff
@@ -5031,6 +5031,43 @@ box_read_ffi_enable(void)
 		box_read_ffi_is_disabled = false;
 }
 
+int
+box_generate_space_id(uint32_t *new_space_id)
+{
+	assert(new_space_id != NULL);
+	static const char empty_key[] = {
```

Comment:
Nit: There's no need to make the buffer static. Also, since this isn't a hot path, it's better to encode MsgPack explicitly instead of using hard-coded constants.

Reply:
Encode it explicitly